src/docs/reference/core/Data/README.md

```diff
@@ -26,27 +26,46 @@ See the [`Microsoft.Data.Sqlite` documentation](https://docs.microsoft.com/en-us
 
 ## Configuring YesSql
 
-OrchardCore uses the `YesSql` library to interact with the configured database provider. `YesSql` is shipped with configuration that is suitable for most use cases. However, you can change these settings by configuring `YesSqlOptions`. `YesSqlOptions` provides the following configurable options
+OrchardCore uses the `YesSql` library to interact with the configured database provider. `YesSql` is shipped with configuration that is suitable for most use cases. However, you can change these settings by configuring `YesSqlOptions`. `YesSqlOptions` provides the following configurable options.
 
 | Setting | Description |
 | --- | --- |
 | `CommandsPageSize` | Gets or sets the command page size. If you have to many queries in one command, `YesSql` will split the large command into multiple commands. |
-| `ContentTypeOptions` | Gets or sets the `QueryGatingEnabled` option in `YesSql`. |
-| `TableNameConvention` | You can provide your own implementation for generating ids. |
+| `QueryGatingEnabled` | Gets or sets the `QueryGatingEnabled` option in `YesSql`. |
+| `IdGenerator` | You can provide your own implementation for generating ids. |
 | `IdentifierAccessorFactory` | You can provide your own value accessor factory. |
 | `VersionAccessorFactory` | You can provide your own version accessor factory. |
 | `ContentSerializer` | You can provide your own content serializer. |
-| `TablePrefixSeparator` | Gets or sets the prefix used to seperate database prefix and the table name. |
 
-For example, you can change the default table prefix seperator from `_` to `__` by adding the following code to your startup code.
+For example, you can change the default command-page-size from `500` to `1000` by adding the following code to your startup code.
 
 ```C#
 services.Configure<YesSqlOptions>(options =>
 {
-    options.TablePrefixSeparator = "__";
+    options.CommandsPageSize = 1000;
 });
 ```
 
+## Database table
+
+The following database table settings, only used as presets before a given tenant is setup, can be provided from any configuration source.
+
+| Setting | Description |
+| --- | --- |
+| `DefaultDocumentTable` | Document table name, defaults to 'Document'. |
+| `DefaultTableNameSeparator` | Table name separator, one or multiple '_', "NULL" means no separator, defaults to '_'. |
+| `DefaultIdentityColumnSize` | Identity column size, 'Int32' or 'Int64', defaults to 'Int64'. |
+
+##### `appsettings.json`
+
+```json
+  "OrchardCore_Data_DatabaseTable": {
```

Comment:
Please refer to my comment about about suggestion to change the name here and the default values.

Reply:
Okay I will look at it